OganesianAA/Lesson 2/homework2.js

```diff
@@ -0,0 +1,78 @@
+const EventEmitter = require('events').EventEmitter;
+const fs = require('fs');
+const readline = require('readline');
+const write = fs.writeFile;
+const util = require('util');
+const read = util.promisify(fs.readFile);
+
+
+class blackJack extends EventEmitter{
```

Comment:
Названия классов с заглавной буквы
```suggestion
class BlackJack extends EventEmitter{
```